CMakeLists.txt

```diff
@@ -37,11 +38,14 @@ add_compile_options(
 add_definitions(-D_LINUX)
 
 add_library(bald_engine
-        library.cpp library.h
+        library.cpp
+        library.h
         src/utils/file_manager.cpp
         src/utils/split_string.cpp
         src/input/input_manager.cpp
-        src/Log.cpp)
+        src/Log.cpp
+        src/graphics/Window.h
```

Comment:
Remove Window.h, it is already included in line 11!

Reply:
fixed